### DISCUSSION TITLE
Getting rejected every time I applied for github student developer pack (renew)

### DISCUSSION THREAD

**Discussion by Kijimu7** [community/community#17808] [GitHub Education]
Hi, I submitted a renewed application for the Student Developer Pack. I was rejected because my student ID doesn’t show the date.
I resubmit my unofficial transcript, which contains the date and shows I am a current student. However, I think I am getting rejected because I cannot take a good quality image. I am using Logitech C920. I tried using my iPhone to get a better-quality picture, but it won’t let me take pictures from my iPhone. There is no button for submitting my unofficial transcript other than taking photos. How can I attach my unofficial transcript showing I am a current student?

**Comment by qweewqi** [comment]
Same here, it asked me to submit from my school location, however I'm getting rejected because the upload button disappeared after the first request full explanation [here](https://github.community/t/somethings-wrong-with-the-request-form/242940) all images taken are blurry, i tried a virtual webcam too with obs the image looks ok once i hit 'take photo' it become Blurred, i don't know what's the problem

**Reply by Kijimu7** [reply]
Hey @qweewqi ! I just got approved! I suggest you go to this link https://support.github.com/contact/education and create a ticket. Tell them what's going on. You will automatically get closed the ticket. However, you can comment and open up the ticket again! Good luck!!

**Reply by qweewqi** [reply]
Thanks  @Kijimu7 i opened a ticket I'm waiting for a reply, fingers crossed 🤞

**Comment by sanafatima8** [comment]
in What is the name of your school? I entered my university name which is also listed below but they didn't process my application instead they send me this message 

----------
GitHub Education benefits are currently only available for accredited degree or diploma granting schools.

Your code school, boot camp, or other organized informal learning institution may apply to become a GitHub Campus Partner:

https://education.github.com/partners/schools

If approved, program participants will become eligible to receive all of the benefits of GitHub Education.

----------

what should I do ? Any suggestions, please?
